components/match2/wikis/rainbow_six/match_summary.lua

```diff
@@ -404,8 +404,29 @@ end
 
 function CustomMatchSummary._createHeader(match)
 	local header = MatchSummary.Header()
-	header  :left(CustomMatchSummary._createOpponent(match.opponents[1], 'left'))
-			:right(CustomMatchSummary._createOpponent(match.opponents[2], 'right'))
+	header  :left(
+				mw.html.create(nil)
+				:node(CustomMatchSummary._createOpponent(match.opponents[1], 'left')
+					:css('width', 'calc(50% - 25px)')
+				)
+				:node(CustomMatchSummary._createScore(match.opponents[1])
+					:css('border-right', '1px solid #aaa')
+					:css('text-align', 'left')
+					:css('width','25px')
+				)
+			)
+			:right(
+				mw.html.create(nil)
+				:node(CustomMatchSummary._createScore(match.opponents[2])
+					:css('border-left', '1px solid #aaa')
+					:css('text-align', 'right')
+					:css('width','25px')
```

Comment:
Always use 4px increments. Make this 
```suggestion
					:css('width','28px')
```

Reply:
Made it 24px instead of 28px.

Reply:
I determined 28px to be a better width in case of 3 digit scores. Is there a particular reason you put 24?

Reply:
Changed it to 28. I thought you picked 28 as an example as it was closest larger divider by 4.